.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: python
+
+matrix:
+  include:
+    - python: 3.7
```

Comment:
Might be worth adding a couple of comments about why we need python 3.7 and xenial in here, even though you explained it well in your PR comments.

Reply:
Good call.